apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/tasks/OperationTasksBuilder.java

```diff
@@ -19,46 +20,47 @@
 @AllArgsConstructor
 public class OperationTasksBuilder {
   @Getter(AccessLevel.NONE)
-  private final OperationTaskFactory<? extends OperationTask> taskFactory;
+  private final OperationTaskFactory<? extends OperationTask<?>> taskFactory;
 
   protected final TablesClient tablesClient;
 
-  private List<OperationTask> prepareTableOperationTaskList(JobConf.JobTypeEnum jobType) {
-    List<OperationTask> taskList = new ArrayList<>();
-    for (TableMetadata metadata : tablesClient.getTables()) {
-      log.info("metadata: {}", metadata);
-      try {
-        taskList.add(taskFactory.create(metadata));
-      } catch (Exception e) {
-        throw new RuntimeException(
-            String.format(
-                "Cannot create operation task from TableMetadata: %s given job type: %s",
-                metadata, jobType),
-            e);
-      }
-    }
-    return taskList;
+  private List<OperationTask<?>> prepareTableOperationTaskList(JobConf.JobTypeEnum jobType) {
```

Comment:
Function descriptions needed.

Reply:
I prefer not adding comments just for the sake of having comments, rather structure code so that it's self-explanatory, but I understand that there might be bias when you build codebase knowledge. Let me know if you think this method would benefit from a docstring, and what information should be in it.

Reply:
Added comments.